src/components/EditDeliveryDetailsPopup/EditDeliveryDetailsPopup.tsx

```diff
@@ -186,29 +190,58 @@ const EditDeliveryDetailsPopup = () => {
       )}
     >
       <ContentStyled>
-        <ThankYouPageStyled>
-          <CheckmarkIcon />
-          <HeaderStyled>
-            {t(
-              'edit-delivery-details-popup.thank-you-page.header',
-              'Delivery Details Updated'
-            )}
-          </HeaderStyled>
-          <InfoTextStyled>
-            {t(
-              'edit-delivery-details-popup.thank-you-page.info-text-1',
-              'Thank you for updating your delivery details for your'
-            )}
-            <p>{t(`offer-title-${offerId}`, offerTitle)}</p>
-            {t(
-              'edit-delivery-details-popup.thank-you-page.info-text-2',
-              'Your changes have been saved and will be reflected in your next delivery.'
-            )}
-          </InfoTextStyled>
-          <Button theme="confirm" onClickFn={() => dispatch(hidePopup())}>
-            {t('edit-delivery-details-popup.button.back', 'Back to settings')}
-          </Button>
-        </ThankYouPageStyled>
+        {error ? (
+          <ThankYouPageStyled>
```

Comment:
I had a bit of a naming dilemma here. Not entirely sure if it's best to keep the ThankYouPageStyled as it is, and change only the content based on the error value, or if we should create a new identical div named ErrorPageStyled to avoid any confusion. 

Let me know your thoughts🤔

Reply:
You can try to do something like that in `EditDeliveryPopupStyled.ts`: 
`export const ErrorPageStyled = styled(ThankYouPageStyled);`
I'm not sure if this syntax is correct but you know what I mean.

But this component is a bit complex now so you can also extract `ErrorPageStyled` to solo component in `EditDeliveryDetailsPopup` folder and just import it here, as I see it doesn't use any variables besides `hidePopup` func and translations so it should be fairly easy.

Reply:
@m4tewoosh changed applied. I've just added ErrorPageStyled that extends the ThankYouPageStyled. Should be ok, I think.